package.json

```
@@ -1,12 +1,12 @@
 {
     "author": {
-        "name": "Edrean Ernst"
+        "name": "Edrean Ernst;Pedro Torrezão"
```

"author" field should include only one person, while "contributors" may be an array of persons
```suggestion
        "name": "Edrean Ernst"
```